dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -72,29 +69,17 @@ def main(project: str,
 
     model = DemoTransformer(vocab_size=dataset.vocab_size,
                             nlayers=int(os.environ.get("NUM_LAYERS", 10)))
-    # Save once per step, and if `save_only_latest`, replace the last checkpoint each time.
-    # Replacing is implemented by saving the new checkpoint, and then deleting the previous one.
-    # If `save_only_latest` is False, a new checkpoint is created for each step.
-    checkpoint_callback = ModelCheckpoint(
-        save_top_k=1 if save_only_latest else -1,
-        every_n_train_steps=1,
-        filename="checkpoint-{epoch:02d}-{step:02d}",
-        enable_version_counter=True,
-    )
-
     strategy = get_strategy(args.strategy, model, ckpt_dir_path)
-    min_epochs_save = int(os.environ.get("MIN_EPOCHS_SAVE", 4))
-    max_epochs_save = int(os.environ.get("MAX_EPOCHS_SAVE", 5))
```

Comment:
Can you update the readme to remove these as well? And possibly the deployment yaml file too if they're set there

Reply:
These vars are not set in the deployment file. I renamed `MAX_STEPS_SAVE` and `MAX_STEPS_RESTORE` to `NUM_SAVE_CALLS` and `NUM_LOAD_CALLS` and updated the README. Do we really need to have these variables configurable?